### PR TITLE
fix return_type of intrinsic functions

### PR DESCRIPTION
**Pull request by gxyd**
fixes integration_tests/intrinsics_16.f90

### CODE REVIEW

File: src/libasr/intrinsic_func_registry_util_gen.py [hunk 1]
```diff
@@ -1011,7 +1011,7 @@ def add_create_func_return_src(func_name):
         src += indent * 4 + "ASR::dimension_t* m_dims = nullptr;\n"
         src += indent * 4 + "size_t n_dims = ASRUtils::extract_dimensions_from_ttype(type, m_dims);\n"
         src += indent * 4 + "return_type = ASRUtils::make_Array_t_util(al, type->base.loc, "
-        src += f"{ret_type}, m_dims, n_dims, ASR::abiType::Source, false, "
+        src += f"return_type, m_dims, n_dims, ASR::abiType::Source, false, "
```
Comment (HarshitaKalani) [inline src/libasr/intrinsic_func_registry_util_gen.py:1014]: I think we don't need this `f` now.
```suggestion
        src += "return_type, m_dims, n_dims, ASR::abiType::Source, false, "
```

**Review by HarshitaKalani — COMMENTED** [review]
Looks good, just a minor comment.

**Comment by gxyd** [comment]
Ah yes, you are right about that. Thanks @HarshitaKalani for the review.

**Comment by certik** [comment]
The CI does not run, we should not be merging PRs until the CI runs.

**Pull request merged**